jetty/content.md

```diff
@@ -8,7 +8,7 @@ Jetty is a pure Java-based HTTP (Web) server and Java Servlet container. While W
 
 # How to use this image.
 
-Run the default Jetty server (`CMD ["jetty.sh", "run"]`):
+Run the default Jetty server (`CMD ["java","-Djava.io.tmpdir=/tmp/jetty","-jar","/usr/local/jetty/start.jar"]`):
```

Comment:
I think I'd just drop the `CMD` here. I think I originally had this because I had copied the `tomcat` docs.